core/video_options.hpp

```diff
@@ -158,6 +158,8 @@ struct VideoOptions : public Options
 			("av-sync", value<std::string>(&av_sync_)->default_value("0us"),
 			 "Add a time offset (in microseconds if no units provided) to the audio stream, relative to the video stream. "
 			 "The offset value can be either positive or negative.")
+			 ("sync", value<std::string>(&sync_)->default_value("off"),
+			  "Whether to synchronise with another camera. Use \"off\", \"server\" or \"client\".")
```

Comment:
Outside the ``#if`` block as well... sorry I missed this one.

Reply:
Thanks!